pkgs/tools/misc/tfk8s/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "tfk8s";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jrhouston";
+    repo = "tfk8s";
+    rev = version;
+    sha256 = "sha256-JzTWbkICOIfsHgMvpXz4bIWaXKKDAD8INSorMvnXiBw=";
+  };
+
+  vendorSha256 = "sha256-r0c3y+nRc/hCTAT31DasQjxZN86BT8jnJmsLM7Ugrq4=";
+  doCheck = false;
+  runVend = true;
+
+  buildFlagsArray = [
+    "-ldflags="
+    "-s"
+    "-w"
+    "-X main.toolVersion=${version}"
+    "-X main.builtBy=nixpkgs"
```

```suggestion
    "-X main.builtBy=NixOS"
```
maybe fits better but either is fine.

---

What if the person is not using NixOS but Nix in another distribution? Shouldn't it be nixpkgs? Because it is a nixpkg and not a NixOS tied package?

---

I have copied this from another package. The repository build system that is building the package is nixpkgs. Let's leave it as nixpkgs?

---

Coping from another package does not mean it is right. nixpkgs is big and there are lots of old packages which needs updating but we can leave it like this.